app/frontend/schools/coaches/components/SA_Coaches_SchoolIndex.res

```diff
@@ -12,82 +14,163 @@ type formVisible =
+               ${isExitedTabSelected ? "" : "border-b-2 border-primary-500 text-primary-500"}`}>
+              <span> {tr("active_coaches")->str} </span>
+              {switch isExitedTabSelected {
+              | true => React.null
+              | false =>
+                <span className=`bg-primary-500 text-white text-xs rounded-md px-1.5 py-1`>
+                  {state.coachCount->Belt.Int.toString->str}
+                </span>
+              }}
+            </a>
+            <a
+              role="tab"
+              href={`${currentPath}archived`}
+              className={`flex gap-1.5 px-5 py-2 items-center p-2 font-medium hover:text-primary-500
+               ${isExitedTabSelected ? "border-b-2 border-primary-500 text-primary-500" : ""}`}>
+              <span className="sm:inline"> {tr("exited_coaches")->str} </span>
+              {switch isExitedTabSelected {
+              | true =>
+                <span className=`bg-primary-500 text-white text-xs rounded-md px-1.5 py-1`>
+                  {state.coachCount->Belt.Int.toString->str}
+                </span>
+              | false => React.null
+              }}
+            </a>
+          </div>
+        </div>
       </div>
       <div className="px-6 pb-4 mt-5 flex flex-1">
         <div className="max-w-2xl w-full mx-auto relative">
-          {state.coaches
-          |> List.sort((x, y) => int_of_string(Coach.id(x)) - int_of_string(Coach.id(y)))
-          |> List.map(coach =>
-            <div
-              key={coach->Coach.id}
-              className="flex items-center shadow bg-white rounded-lg mb-4 overflow-hidden">
-              <div className="course-faculty__list-item flex w-full">
-                <button
-                  ariaLabel={"Edit: " ++ (coach |> Coach.name)}
-                  className="course-faculty__list-item-details flex flex-1 items-center justify-between cursor-pointer rounded-lg hover:bg-gray-50 hover:text-primary-500 focus:outline-none focus:text-primary-500 focus:bg-gray-50 focus:ring-2 focus:ring-inset focus:ring-focusColor-500"
-                  onClick={_event => {
-                    ReactEvent.Mouse.preventDefault(_event)
-                    send(UpdateFormVisible(CoachEditor(Some(coach))))
-                  }}>
-                  <div className="flex flex-1 py-4 px-4">
-                    <img
-                      className="w-10 h-10 rounded-full me-4 object-cover"
-                      src={coach |> Coach.imageUrl}
-                      alt={tr("avatar_of") ++ (coach |> Coach.name)}
-                    />
-                    <div className="text-sm ">
-                      <p className="font-semibold"> {coach |> Coach.name |> str} </p>
-                      <p className="text-gray-600 text-xs mt-px"> {coach |> Coach.title |> str} </p>
+          {switch state.coaches->Belt.List.some(coach =>
+            Coach.exited(coach) == isExitedTabSelected
```

Comment:
Most of this complex logic comes up as we are not simply reloading the page on archive 😅

Reply:
Yes!